packages/components/src/templates/next/components/complex/InfoCols/InfoCols.tsx

```diff
@@ -71,10 +63,10 @@ const InfoCols = ({
 }: InfoColsProps) => {
   return (
     <section className="bg-white">
-      <div className={`${ComponentContent} py-24`}>
-        <div className="flex flex-col gap-24">
+      <div className={`${ComponentContent} py-12 md:py-16`}>
+        <div className="flex flex-col gap-12">
           <InfoColsHeader title={title} subtitle={subtitle} />
-          <InfoBoxes infoBoxes={infoBoxes} LinkComponent={LinkComponent} />
+          <InfoBoxes infoBoxes={infoBoxes} />
```

Comment:
should we still pass this in so it can be used inside as the link component to replace the `a` tag

Reply:
Addressed in https://github.com/opengovsg/isomer/pull/432/commits/d56f31135b69208dd6a8d17e0ebcb934e3d04dce!